content/en/agent/logs/advanced_log_collection.md

```diff
@@ -746,15 +746,17 @@ logs:
 The example above matches `C:\\MyApp\\MyLog.log` and excludes `C:\\MyApp\\MyLog.20230101.log` and `C:\\MyApp\\MyLog.20230102.log`.
 
 **Note**: The Agent requires read and execute permissions on a directory to list all the available files in it.
-**Note2**: The path and exclude_paths values are case sensitive.
 
-## Tail most recently modified files first
+**Note**: The path and exclude_paths values are case sensitive.
+
+### Tail most recently modified files first
```

Comment:
Changed the heading level from ## to ### to make it a subsection of `Tail directories using wildcards`.